### ISSUE TITLE
[description] Comparison of repos

### ISSUE THREAD

**Issue by DonaldTsang** [n4n0GH/0xchan-frontend#42]
What is the difference, feature and design wise, between 0xchan and these other repos?
- https://github.com/yatima1460/Kamina (IPFS)
- https://github.com/majestrate/nntpchan (NNTP)
- https://github.com/smugdev/smugboard (IPFS)
- Other broken repo
  - https://github.com/0x10F/dchan
  - https://github.com/marios8543/Blockchan-prototype

**Comment by n4n0GH** [comment]
Did you read the 0xchan whitepaper and did you do your research on the boards you want to compare?

**Comment by DonaldTsang** [comment]
@n4n0GH there is a whitepaper? If so please link it in the ReadME. Apologies, but it would be a good idea to have both an ELI5 of the white paper, and a comparison of other similar services on the ReadME.

**Comment by n4n0GH** [comment]
An ELI5 is not the scope of a project that is set behind higher entry barriers to usage. And I will not redundantly add information to the readme, just browse my repositories as they are all clearly labelled.

**Comment by DonaldTsang** [comment]
The non-inclusion of ELI5 is understandable, but what about a comparison between a ETH blockchain-based imageboard with IPFS, and a NNTP-based or pure IPFS-based imageboard (regarding PROs and CONs)?

**Comment by n4n0GH** [comment]
What would doing that solve? Why should I spend time working on that?

**Comment by DonaldTsang** [comment]
@n4n0GH possibly because it would be beneficial to "selling the product" or "convincing the consumer"? There seems to be no media kits that allows people to understand what you are creating and providing, and media kits are common for many projects, no matter cultural or economic or technological.

**Comment by n4n0GH** [comment]
But I don't need to "sell the product" or convince anyone to use this. Your logic is flawed and you have proven to not know a single thing about this project. Please read up on everything over at 0xchan.net before opening more issues.

**Comment by DonaldTsang** [comment]
@n4n0GH what? I mean "using blockchain (smart contracts) to create an experimentally uncensorable cloud imageboard" is pretty much a one-liner with too little on how it is advantageous when compared to other existing products since it makes no comparisons? And of course is this circumventing IPFS's Filecoin system?

**Comment by n4n0GH** [comment]
Again: I don't need to compare this to other projects.
And IPFS is not mandating the use of Filecoin at all.
Please, read the whitepaper, read the info on the website and make informed questions. I see no purpose in discussing things with someone who clearly just wants to be spoonfed.
